RecoPixelVertexing/PixelLowPtUtilities/interface/StripSubClusterShapeTrajectoryFilter.h

```diff
@@ -40,6 +42,12 @@ class StripSubClusterShapeFilterBase {
                    const GlobalVector &gdir,
                    bool mustProject = false) const;
 
+  // esConsumes tokens
+  edm::ESGetToken<TrackerTopology, TrackerTopologyRcd> m_topoToken;
+  edm::ESGetToken<ClusterShapeHitFilter, CkfComponentsRecord> m_csfToken;
+  edm::ESGetToken<TrackerGeometry, TrackerDigiGeometryRecord> m_geomToken;
+  edm::ESGetToken<SiStripNoises, SiStripNoisesRcd> m_stripNoiseToken;
```

Comment:
```suggestion
  edm::ESGetToken<TrackerTopology, TrackerTopologyRcd> topoToken_;
  edm::ESGetToken<ClusterShapeHitFilter, CkfComponentsRecord> csfToken_;
  edm::ESGetToken<TrackerGeometry, TrackerDigiGeometryRecord> geomToken_;
  edm::ESGetToken<SiStripNoises, SiStripNoisesRcd> stripNoiseToken_;
```
would this be acceptable here, to stick with the trailing underscore as a member identifier?

Reply:
done.